src/gateways/createUserLoader.spec.ts

```diff
@@ -28,36 +28,31 @@ describe('userLoader', () => {
         (global as any).fetch = mockFetch(200, testAccountsUser);
 
         const user = await userLoader.getCurrentUser();
-        expect(fetch).toHaveBeenCalledWith('url/api/user', {
+        expect(fetch).toHaveBeenCalledWith('url/accounts/api/user', {
           credentials: 'include',
         });
         expect(user).toEqual(testAccountsUser);
       });
 
-      it('gets undefined', async() => {
+      it('gets undefined for 403 errors', async() => {
         (global as any).fetch = mockFetch(403, '');
 
         const user = await userLoader.getCurrentUser();
-        expect(fetch).toHaveBeenCalledWith('url/api/user', {
+        expect(fetch).toHaveBeenCalledWith('url/accounts/api/user', {
           credentials: 'include',
         });
         expect(user).toEqual(undefined);
       });
     });
 
-    it('throws for unexpected errors', async() => {
+    it('gets undefined for unexpected errors', async() => {
```

Comment:
Different behavior here: browserAuthProvider returns undefined for all non-200 status codes, not just 403.

Reply:
i think the rex behavior of throwing for other statuses is probably more correct, we could update the browserProvider to match

Reply:
Cool, I'll update the provider